LanguageFeatures/Augmentation-libraries/augmenting_constructors_A04_t02_lib.dart

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion It is a compile-time error if:
+/// ...
+/// - The introductory constructor is marked `factory` and the augmenting
+/// constructor is not, or vice versa.
+///
+/// @description Checks that it is a compile-time error if the introductory
+/// constructor is marked `factory` and the augmenting constructor is not.
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=macros
+
+augment library 'augmenting_constructors_A04_t02.dart';
+
+augment class C {
+  augment C(): super(0);
```

Comment:
This should be a `factory` constructor, right?

Reply:
Right, thank you!